templates/deployment.yaml

```diff
@@ -569,6 +595,7 @@ spec:
         - name: bird-socket
           mountPath: /var/run/bird/
       {{- end }}
+      {{- end }}
```

Comment:
Where is the start to this new end?

Reply:
https://github.com/jukrut/cgw/blob/feature/bird-reload/templates/deployment.yaml#L530